.gitignore

```diff
@@ -6,3 +6,17 @@ lib
 .nyc_output
 coverage
 es
+test/**/lcov.info
+test/**/lcov-report
+test/react/0.14/test/components/*.spec.js
+test/react/0.14/test/utils/*.spec.js
+test/react/15/test/components/*.spec.js
+test/react/15/test/utils/*.spec.js
+test/react/16.2/test/components/*.spec.js
+test/react/16.2/test/utils/*.spec.js
+test/react/16.3/test/components/*.spec.js
+test/react/16.3/test/utils/*.spec.js
+test/react/16.4/test/components/*.spec.js
+test/react/16.4/test/utils/*.spec.js
```

Comment:
You can wildcard this (`test/react/*/test/**/*.spec.js`) so you don't have to list every subdirectory out or maintain it with newer versions of React.